zmon_agent/discovery/kubernetes/cluster.py

```diff
@@ -639,5 +649,27 @@ def get_postgresql_databases(postgresql_clusters, cluster_id, alias, environment
                 'database_name': db,
                 'shards': {
                     db: '{}:{}/{}'.format(pgcluster['dnsname'], POSTGRESQL_DEFAULT_PORT, db)
-                }
+                },
+                'role': 'master'
             }
+
+            if pgcluster['expected_replica_count'] > 1:  # the first k8s replica is the master itself
+                name_parts = pgcluster['dnsname'].split('.')
+                repl_dnsname = '.'.join([name_parts[0] + '-repl'] + name_parts[1:])
+                yield {
+                    'id': '{}-repl-{}'.format(db, pgcluster['id']),
+                    'type': POSTGRESQL_DATABASE_REPLICA_TYPE,
+                    'kube_cluster': cluster_id,
+                    'alias': alias,
+                    'environment': environment,
+                    'created_by': AGENT_TYPE,
+                    'infrastructure_account': infrastructure_account,
+                    'region': region,
+
```

Comment:
missing application and version field

Reply:
I don't really think application adds here anything.  There is no `postgresql_database` entity with other application.